bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/calunit/CalUnitParams.java

```diff
@@ -124,6 +125,10 @@ public static boolean containsArc(Collection<Lamp> lamps) {
             String[] ar = formattedList.split(",");
             return Arrays.stream(ar).map(Lamp::getLamp).collect(Collectors.toList());
         }
+
+        public static String write(final Set<Lamp> lamps) {
+            return show(lamps, Lamp::name);
```

Comment:
Since they're stored as a `TreeSet` in the only object that calls this method, I changed this to a `TreeSet` to be safe. I don't think order matters (as we're just making into a comma-separated string) but why tempt the fates?